### ISSUE TITLE
Accessibility: missing tab navigation support for night shift toggle button and focus indicator styles for few other elements inside sidebar

### ISSUE THREAD

**Issue by heybran** [TryGhost/Ghost#25202]
#### 1. Missing tab navigation support
The night shift toggle button appears to be inaccessible to tab navigation due to its missing `tabindex` attribute.

#### 2. Missing tab focus indicator style, therefore users are not aware of which element is currently being focused
1. Search button (below screenshot is when it's currently being focused by tab)

<img width="319" height="142" alt="Image" src="https://github.com/user-attachments/assets/9f86558d-8fab-465c-bd1d-33471cba0ec0" />

---

2. Toggle sidebar button (below screenshot is when it's currently being focused by tab)

<img width="347" height="273" alt="Image" src="https://github.com/user-attachments/assets/822b3e19-3aac-43bf-9010-d8a229c1ffac" />

---

3. Open site in new tab link (below screenshot is when it's currently being focused by tab)

<img width="321" height="235" alt="Image" src="https://github.com/user-attachments/assets/96e240be-64b9-404d-a9c7-e0da9f02c009" />

---

4. Toggle custom post types button (below screenshot is when it's currently being focused by tab)

<img width="316" height="327" alt="Image" src="https://github.com/user-attachments/assets/688fc5a4-0816-414b-9fe8-93f52deb0c12" />

---

5. User avatar button (below screenshot is when it's currently being focused by tab)

<img width="317" height="282" alt="Image" src="https://github.com/user-attachments/assets/934ca96c-aecd-4644-918b-781d2257feac" />

### Steps to Reproduce

1. Open the Ghost admin
2. Attempt to tab through the items inside the sidebar

### Ghost Version

6.4.0

### Node.js Version

22.19.0

### How did you install Ghost?

CLI

### Database type

MySQL 8

### Browser & OS version

Brave browser & macOS 15.3

### Relevant log / error output

```shell

```

### Code of Conduct

- [x] I agree to be friendly and polite to people in this repository

**Comment by abdultalha0862** [comment]
@heybran Thank you for raising this issue. Currently, I am working on this issue.

**Comment by abdultalha0862** [comment]
@heybran Can you provide a video so that it will be clear to the contributors to effectively understand the issue and work on it.

**Comment by heybran** [comment]
Hey @abdultalha0862 

If you try to tab though the items in the sidebar, you will find that for the items I listed above, they need a styles applied to their focus state, in order for the users to know which UI element is focused at the moment. It's the same idea when you navigate the website using screen reader, the screen reader will announce the element back to you, so you know which actions to perform.

If it's still not clear, I can upload a video when I got a bit of more time.

**Comment by abdultalha0862** [comment]
@heybran I am grateful for your detailed explanation. I understand you may be busy, and if possible, I would appreciate it if you could share the video or other resources. Otherwise, I will do my best to understand the issue.

**Comment by heybran** [comment]
Hey @abdultalha0862,

Sorry for the late reply, I've recorded a short screencast explaining about the issue, only to realize that you've already submitted the PR. 

In case you need to watch my screencast, here's the link to the video: https://youtu.be/Ijaz_1BRVUo It's 43M, so I uploaded to Youtube.

**Comment by jagadish-7** [comment]
Hi @heybran @abdultalha0862  is this issue still open? I would love to take this up.

**Comment by heybran** [comment]
Hi @jagadish-7,

I'm not quite sure what's going on any more. I see that the PR created by @abdultalha0862 is closed and no one is actually reviewing his PR excerpt some AI bot review. So it might be difficult to work on something like this. But let's see how @abdultalha0862 responses, I might be wrong.
